src/styles/common/_typography.scss

```diff
@@ -19,6 +19,10 @@ body {
     @include common-typography;
 }
 
+.Body.is-deja-blue {
```

Comment:
Check that this also affects modals and other detached roots

Reply:
modals/portalled elements have this new font size